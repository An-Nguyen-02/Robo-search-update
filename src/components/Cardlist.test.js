import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Cardlist from './Cardlist';

it('card list snapshot',()=>{
	const mockRobots = [
		{
			id: 1,
			name: 'hello',
			username: 'hihi',
			email: 'hello@gmail.com'
		}
	]
	expect(shallow(<Cardlist robots={mockRobots}/>)).toMatchSnapshot();
})