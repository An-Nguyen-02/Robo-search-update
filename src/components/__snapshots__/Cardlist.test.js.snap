// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card list snapshot 1`] = `
<Fragment>
  <Card
    email="hello@gmail.com"
    id={1}
    key="1"
    name="hello"
  />
</Fragment>
`;
